Makefile

```diff
@@ -24,6 +24,15 @@ BUILD_DATE := $(shell date -u +'%Y-%m-%dT%H:%M:%SZ')
 ## Absolute or relative GOPATH for the 'build' target
 BUILD_DIR ?= build
 BUILD_DIR_ABS := $(abspath $(BUILD_DIR))
+ifeq ($(DOCKER_BUILD_CACHE),true)
+DOCKER_BUILD_FLAGS += --no-cache=false
+else ifeq ($(DOCKER_BUILD_CACHE),false)
+DOCKER_BUILD_FLAGS += --no-cache=true
+else ifneq ($(DOCKER_BUILD_CACHE),)
+$(error DOCKER_BUILD_CACHE should be either true or false)
+else
+DOCKER_BUILD_FLAGS += --no-cache=true
+endif
```

Comment:
Probably, we can make this block a bit simpler? Here we also add a comment for adjustable variable (which is expected since we started creating self-documenting Makefiles).
The comment explicitly describes accepted variables and minimizes the risk of mistake.
```suggestion
## Enable Docker build cache (true|false)
DOCKER_BUILD_CACHE ?= false
ifeq ($(DOCKER_BUILD_CACHE),true)
DOCKER_BUILD_FLAGS += --no-cache=false
else
DOCKER_BUILD_FLAGS += --no-cache=true
endif
```